app/helpers/application_cocoon_helper.rb

```diff
@@ -0,0 +1,55 @@
+module ApplicationCocoonHelper
+  # Shows a link that will allow to dynamically add a new associated object. This is a wrapper of
+  #   {#link_to_add_association}
```

Comment:
yikes, I didn't see this. Is this syntax equivalent to `+link_to_add_association+`?